README-DEV.md

```diff
@@ -2,137 +2,33 @@
-
-#### Config Tag Specific Checks: `schedule-2-start.yml`
-
-##### `config`
-Similar to the [`pr-1-ci.yml` counterpart](#config) - parses the CI configuration file for markers, `model-config-tests` and python versions for scheduled reproducibility checks.
-
-##### `repro-ci`
-
-Exactly the same as the [`pr-1-ci.yml` counterpart](#repro-ci) - run the config associated with the given config tag and upload the checksum.
-
-##### `check-repro`
-
-Similar to the [`pr-1-ci.yml` counterpart](#check-repro), this job checks the recently generated checksum against the existing config tag - essentially a self-test.
-
-##### `failed-repro`
-
-If the reproduction check fails during a scheduled check, something is wrong. This job creates an issue in this repository, and tags the Release team for investigation.
-
-### Supplementary Workflows
-
-#### Config Validation: `validate-json.yml`
-
-This workflow is used to check that modifications to `*.json` files are in line with the associated `*.schema.json`.
-
-#### Initial Checksum Creation: `generate-initial-checksums.yml`
-
-This `workflow_dispatch`-triggered workflow generates checksums of a given config branch, and optionally commits them. This is useful for generating checksums for an entirely new config branch, so the workflows above have something to compare against.
-
-### CI Configuration File
+### CI Configuration
 
 This is the `config/ci.json` configuration file for specifying different test markers, or test versions based on type of the test to run, and the name of the git branch or tag. The different types of test are defined as:
-- `scheduled`: These are scheduled monthly reproducibility tests that are run as part of [`schedule-2-start.yml`](#repro-ci-1). The keys under these tests represent released config tags to run scheduled checks on.
-- `reproducibility`: These are reproducibility tests are run as part of [`pr-1-ci.yml`](#repro-ci). The keys under these tests represent the target branches into which pull requests are being merged.
-- `qa` - These are quick QA tests are run as part of [`pr-1-ci.yml`](#qa-ci). The keys under these tests represent the target branches into which pull requests are being merged.
+
+- `scheduled`: These are scheduled monthly reproducibility tests that are run as part of [`schedule-2-start.yml`](#scheduled). The keys under these tests represent released config tags to run scheduled checks on.
+- `reproducibility`: These are reproducibility tests are run as part of [`pr-1-ci.yml`](#pull-request). The keys under these tests represent the target branches into which pull requests are being merged.
+- `qa` - These are quick QA tests are run as part of [`pr-1-ci.yml`](#pull-request). The keys under these tests represent the target branches into which pull requests are being merged.
```

Comment:
```suggestion
- `scheduled`: Scheduled monthly reproducibility tests. The keys under these tests represent released config tags to run scheduled checks on.
- `reproducibility`: Reproducibility tests that are run as part of pull requests. The keys under these tests represent the target branches into which pull requests are being merged.
- `qa` - Quick quality assurance tests that are run as part of pull requests. The keys under these tests represent the target branches into which pull requests are being merged.
```
Probably don't need to refer the the workflow names anymore as they aren't mentioned in this README